### PR TITLE
Sak 40520

### PR DESCRIPTION
**Pull request by RyanAFinney**

### CODE REVIEW

File: site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties [hunk 1]
```diff
@@ -1029,3 +1030,49 @@ gen.listnavselect = To operate the combo box, first press Alt+Down Arrow to open
 search.label=Search
 search.button=Search
 search.clear=Clear Search
+
+help-tips-header=While you wait, enjoy some cheer from the Isidore Cheer Cup!
```
Comment (fostersdesign) [inline site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties]: These help tips contain an institutional-specific branding of Sakai. The lines should dynamically pull in the local Sakai name from the "ui.service" from sakai.properties.

File: site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties [hunk 2]
```diff
@@ -1029,3 +1030,49 @@ gen.listnavselect = To operate the combo box, first press Alt+Down Arrow to open
...
+help-tips-favorite-header=Favoriting Isidore Course Sites
+help-tips-favorite=Click on the star next to course sites you want to add to your favorites. Favoriting these sites will pin them to the top blue menu bar in Isidore.
```
Comment (fostersdesign) [inline site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties]: What does the help tips for favouriting sites have to do with the Overview page? Why is it included in this PR?

File: portal/portal-service-impl/impl/src/sql/mysql/bullhorn_tables.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
+CREATE TABLE BULLHORN_ALERTS
```
Comment (fostersdesign) [inline portal/portal-service-impl/impl/src/sql/mysql/bullhorn_tables.sql]: Why is the creation of Bullhorn tables included in this PR?

File: portal/portal-api/api/src/java/org/sakaiproject/portal/api/BullhornService.java [hunk 4]
```diff
@@ -85,4 +85,21 @@
...
     public boolean clearAllAcademicAlerts(String userId);
+
+    /**
+     * Toggles the bullhorn alert flag to be shown or not by saving
+     * a boolean in the user preferences table.
+     *
+     * @param userId, User Id to toggle
+     * @return The value of hidden preference after save
+     */
+    boolean toggleHideBullhornAlerts(String userId);
```
Comment (fostersdesign) [inline portal/portal-api/api/src/java/org/sakaiproject/portal/api/BullhornService.java]: Why are these Bullhorn lines included in this PR about overview?

File: library/src/morpheus-master/sass/modules/navigation/_skipnav.scss [hunk 5]
```diff
@@ -93,6 +93,17 @@
 				
 				a.#{$namespace}skipNav--toolName {
 					color: $link-color;
+				}
+				a.manage-overview-link-mobile {
+					color: $link-color;
+					cursor: pointer;
+					display: inline-block;
+					font-family: "Open Sans", sans-serif;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/navigation/_skipnav.scss]: Can these font family declarations be pulled from the default font-family variable in SASS? Are they not cascading from parent elements anyways?

File: library/src/morpheus-master/sass/modules/navigation/_base.scss [hunk 6]
```diff
@@ -1277,3 +1277,34 @@ body.is-logged-out{
...
+		border: 1px solid #ccc;
+		color: #333 !important;
+		cursor: pointer;
+		display: inline-block;
+		font-family: "Open Sans", sans-serif;
+		font-size: 12px;
+		text-decoration: none !important;
+		-moz-appearance: none;
+		-webkit-appearance: none;
+		-moz-border-radius: 3px;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/navigation/_base.scss]: There is a mixin' for border-radius: @include border-radius(3px); that will append the vendor prefixes.

File: library/src/morpheus-master/sass/modules/navigation/_base.scss [hunk 7]
```diff
@@ -1277,3 +1277,34 @@ body.is-logged-out{
...
+@media only screen and (min-width: 801px) {
+	.manage-overview-link {
+		border: 1px solid #ccc;
+		color: #333 !important;
+		cursor: pointer;
+		display: inline-block;
+		font-family: "Open Sans", sans-serif;
+		font-size: 12px;
+		text-decoration: none !important;
+		-moz-appearance: none;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/navigation/_base.scss]: There is a mixin' for appearance: @include appearance(none) that will append the vendor prefixes.

File: library/src/morpheus-master/sass/modules/navigation/_base.scss [hunk 8]
```diff
@@ -1277,3 +1277,34 @@ body.is-logged-out{
         }
     }
 }
+
+@media only screen and (min-width: 801px) {
+	.manage-overview-link {
+		border: 1px solid #ccc;
+		color: #333 !important;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/navigation/_base.scss]: Does this color have to have "!important"?

File: library/src/morpheus-master/sass/modules/bullhorn-alerts/_base.scss [hunk 9]
```diff
@@ -21,15 +21,20 @@
 }
 
 .portal-bullhorns.qtip {
-    border: 0 none;
-    box-shadow: 0 0 4px rgba( $text-color ,0.35);
+    border: 1px solid #E0E0E0;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/bullhorn-alerts/_base.scss]: Why are these Bullhorn changes in a PR about Overview?

File: library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js [hunk 10]
```diff
@@ -224,6 +261,28 @@
         });
     });
 
+    portal.hideShowAlerts = function () {
```
Comment (fostersdesign) [inline library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js]: Why are these Bullhorn functions included in this PR about the Overview page?

File: library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js [hunk 11]
```diff
@@ -207,12 +244,12 @@
 
                                     var time = formatDate(alert.eventDate);
 
-                                    markup += message + '</div><div class="portal-bullhorn-time">' + time + '</div>';
-                                    markup += '</div><div class="portal-bullhorn-clear"><a href="javascript:;" onclick="portal.clearBullhornAlert(\'academic\', \'' + alert.id + '\',\'' + data.i18n.noAlerts + '\');" title="' + data.i18n.clear + '"><i class="fa fa-times" aria-hidden="true"></i></a></div></div></a>';
+                                    markup += message + '</div><div class="portal-bullhorn-site">' + siteTitle + '</div><div class="portal-bullhorn-time">' + time + '</div>';
```
Comment (fostersdesign) [inline library/src/morpheus-master/js/src/sakai.morpheus.bullhorns.js]: Why are these Bullhorn lines included in a PR about the Overview page?

**Comment by RyanAFinney** [comment]
The bullhorn and institutional branding was me working through too fast-- I cherry-picked the wrong commits and didn't review it properly before submitting the PR. Closing and will re-open after reworking.

**Pull request closed without merging**